app/controllers/application_controller.rb

```diff
@@ -1,3 +1,12 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    attributes = [:name, :about]
```

Comment:
fun Ruby thing: you can create an Array of Symbols with the sigil `%i` like so: `%i[name about]`.